lib/project_types/theme/messages/messages.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module Theme
+  module Messages
+    MESSAGES = {
+      ensure_themekit_installed: {
+        downloading: "Downloading Themekit %s",
+        successful: "Themekit installed successfully",
+        unsuccessful: "Unable to verify download digest",
```

Comment:
Is failing to verify the download the only failure state? Or is this message shown in case of any error? (network problems, maybe?) If we don't know we might want this message to be less specific.

Reply:
That's a good point, I think we could also make sure we fail gracefully if the download / JSON fails for whatever reason.